pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Studio AsyncAPI document validation"
+description: This tutorial will teach us how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+# Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+We will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
+
+# Background context
+An AsyncAPI document is a file that defines and annotates the different components of a specific Event-Driven API. The format of the file must be JSON or YAML. You can use this document to generate both documentation and code.
+
+The AsyncAPI Studio tool allows you to develop an AsyncAPI document, validate it, preview it, convert it to the latest version, and visualize event flows.
+
+# Copy invalid AsyncAPI document
+Let's pretend we have an invalid AsyncAPI document.
+
+1. Open [Studio](https://studio.asyncapi.com/).
+
+2. Navigate to the left side bar and click on **new file**.
+
+3. Copy and paste the below invalid AsyncAPI document:
+
+```yaml
+asyncapi: '2.7.0'
+info:
+  title: Account Service
+  version: 1.0.0
+  
+channels:
+  user/signedup:
+    subscribe:
+      message:
+        $ref: 'UsersSignedUp'
+components:
+  messages:
+    UserSignedUp:
+      payload:
+        type: object
+        properties:
+          displayName:
+            type: string
+            description: Name of the user
+          email:
+            type: string
+            format: email
+            description: Email of the user
+ ``` 
+
+# Troubleshoot Studio console errors
+Let's fix the errors one by one until we end up with a valid AsyncAPI document.
+
+1. In Studio, open the **visualizer** found on the left-side menu.
```

Comment:
why do I need to open visualizer to troubleshoot errors?

Reply:
fixed it